### PR TITLE
Adds a blog portion to the website. Modifies spacing around objects.

### PR DESCRIPTION
**Pull request by hailed748**
This adds a lot of elements specifically on the blog portion of the website. Because the changes were primarily on the blog portion and do not impact other parts of the site I did it as one commit. Let me know if this is fine in the future, or if I should still break it down into smaller commits.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -88,5 +87,115 @@ <h1 class ="titles">Photography</h1>
...
+          <a class="card" href="#">
+            <span class="card-thumbnail thumb1">
+              <span><h4 class="card-topic">Update</h4></span>
+              <span class="date">
+                <h3 class="day">17</h3>
+                <h4 class="month">Jun</h4>
+              </span>
+            </span>
+            <span class="card-summary">
+              <h1 class="card-title"> I am a Noogler! </h1>
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:103]: Remove extra spaces

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -88,5 +87,115 @@ <h1 class ="titles">Photography</h1>
         <figure class="cell cell-6"><img class="grid-pic" src="images/photos/p13.jpg"></figure>
       </secion>
     </div>
+    <div class="blog">
+      <h1 class="titles">Blog</h1>
+      <div class="container">
+        <div class="cards">
+          <a class="card" href="#">
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/index.html:94]: It's intentional that you want to scroll to the top of the page when this is clicked, yeah?

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (LukeGMTravis) [reply portfolio/src/main/webapp/index.html:103]: Same below

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -103,7 +103,7 @@ html {
 
 .titles{
     font-size: 48px;
-    padding: 60px 0;
+    padding: 60px 0 20px 0;
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:106]: You can simplify this to `60px 0 20px;`

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -181,4 +183,153 @@ html {
...
+
+.blog{
+    background-color: gainsboro;
+    height: 100vh;
+    padding: 50px 200px;
+}
+
+
+.container {
+    display: block;
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:196]: Nit: Inconsistant indenting

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -181,4 +183,153 @@ html {
...
+
+.card {
+    background: white;
+    border-radius: 10px;
+    color: darkslategrey;
+    margin: 45px;
+    max-width: 380px;
+    position: relative;
+    text-decoration: none;
+    box-shadow: 0 1px 2px rgba(0,0,0,0.15);
```
Comment (LukeGMTravis) [inline portfolio/src/main/webapp/style.css:218]: Nit: Spaces after commas, same below

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (hailed748) [reply portfolio/src/main/webapp/index.html:94]: It is just a dummy link until I write the blog.

**Pull request merged**